src/validation/userValidation.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable require-jsdoc */
+import Validator from 'validatorjs';
+import authService from '../services/authService';
+
+
+/**
+ * UsersValidation
+ */
+
+class UsersValidation {
+  /**
+   * @returns {object} ValidateUserSignIn
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   */
+  static async ValidateUserSignIn(req, res, next) {
+    let { email, password } = req.body;
+    const constraint = {
+      email: 'required|email|min:12|max:30',
```

Comment:
why will you have a `min` and `max` for emails?

Reply:
I will fix that